fluss-client/src/test/java/org/apache/fluss/client/admin/FlussAdminITCase.java

```diff
@@ -1092,20 +1093,33 @@ void testDynamicConfigs() throws ExecutionException, InterruptedException {
         assertConfigEntry(
                 DATALAKE_FORMAT.key(), null, ConfigEntry.ConfigSource.DYNAMIC_SERVER_CONFIG);
 
-        // Delete dynamic configs to use the initial value(from server.yaml)
         admin.alterClusterConfigs(
-                        Collections.singletonList(
+                        Arrays.asList(
+                                new AlterConfig(
+                                        DATALAKE_FORMAT.key(), "paimon", AlterConfigOpType.SET),
                                 new AlterConfig(
-                                        DATALAKE_FORMAT.key(), null, AlterConfigOpType.DELETE)))
+                                        "datalake.paimon.warehouse",
+                                        "test-warehouse",
+                                        AlterConfigOpType.SET)))
                 .get();
-        assertThat(
-                        FLUSS_CLUSTER_EXTENSION
-                                .getCoordinatorServer()
-                                .getCoordinatorService()
-                                .getDataLakeFormat())
-                .isEqualTo(PAIMON);
-        assertConfigEntry(
-                DATALAKE_FORMAT.key(), "paimon", ConfigEntry.ConfigSource.INITIAL_SERVER_CONFIG);
+        TablePath tablePath = TablePath.of("test_db", "test_table");
+        createTable(
+                tablePath,
+                TableDescriptor.builder()
+                        .schema(DEFAULT_SCHEMA)
+                        .property(ConfigOptions.TABLE_DATALAKE_ENABLED, true)
+                        .build(),
+                true);
+
+        waitUntil(
+                () -> {
+                    TableInfo tableInfo = admin.getTableInfo(tablePath).get();
+                    Map<String, String> tableProperties = tableInfo.getProperties().toMap();
+                    return "test-warehouse"
+                            .equals(tableProperties.get("table.datalake.paimon.warehouse"));
```

Comment:
minor: we can check `tableProperties` has key 'datalake.format' and 'table.datalake.paimon.warehouse' firstly and then check value of two properties